pom.xml

```diff
@@ -207,6 +207,17 @@
       </snapshots>
     </repository>
     <repository>
+      <id>bintray</id>
+      <name>Bintray</name>
+      <url>https://dl.bintray.com/readytalk/maven</url>
```

Comment:
So in general, we can't depend on artifacts that aren't in maven central in Spark's build. The issue is that these can be removed later, taken down, etc, and it will break all spark builds in the wild.